app/controllers/SchemeDetailsController.scala

```diff
@@ -74,19 +70,22 @@ class SchemeDetailsController @Inject()(appConfig: FrontendAppConfig,
   }
 
   private def openedDate(srn: String, list: ListOfSchemes, isSchemeOpen: Boolean): Option[String] = {
-      if(isSchemeOpen) {
-        list.schemeDetail.flatMap { listOfSchemes =>
-          val currentScheme = listOfSchemes.filter((i: SchemeDetail) => i.referenceNumber.contains(srn))
-          if (currentScheme.nonEmpty) {
-            currentScheme.head.openDate.map(new LocalDate(_).toString(DateHelper.formatter))
-          } else { None }
+    if (isSchemeOpen) {
+      list.schemeDetail.flatMap { listOfSchemes =>
+        val currentScheme = listOfSchemes.filter((i: SchemeDetail) => i.referenceNumber.contains(srn))
```

Comment:
Could simplify to filter(_.referenceNumber....

Reply:
done